server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/DeleteUserDataRoutes.java

```diff
@@ -73,7 +73,7 @@ public Route deleteUserData() {
             .register(DELETE_USER_DATA, request -> {
                 Username username = Username.of(request.params(USER_PATH_PARAM));
 
-                Preconditions.checkArgument(usersRepository.contains(username), "'username' parameter should be an existing user");
+                Preconditions.checkArgument(request.queryParams("force") != null || usersRepository.contains(username), "'username' parameter should be an existing user");
```

Comment:
`force` = `true` is better?

Reply:
Why? `?force` is enough, IMO no need to add a dummy boolean value to it.